src/error_chain.rs

```diff
@@ -188,7 +188,12 @@ macro_rules! impl_error_chain_processed {
                             $(
                                 $(#[$meta_foreign_links])*
                                 $error_kind_name::$foreign_link_variant(ref foreign_err) => {
-                                    foreign_err.cause()
+                                    match () {
```

Comment:
`match` here is unnecessary, can be replaced with
```rust
                                    #[cfg(not(has_error_source))]
                                    {
                                        foreign_err.cause()
                                    }
                                    #[cfg(has_error_source)]
                                    {
                                        foreign_err.source()
                                    }
```

Reply:
this approach with custom cfg flag will not work in dependent code since there will be not such flag defined and compilation will fail

Reply:
thats what we addressed with @prekucki in last commits here

Reply:
@hcpl pls reply

Reply:
Sorry, I thought that my involvement here would be unnecessary because the review comment became outdated. And yes, I forgot that in my solution the flag is processed after `error-chain` has been compiled and the macro has been expanded, my bad.

Do I need to do anything now?

Reply:
so pls resolve commend below

Reply:
I can't find how I can do that. Someone with write access will be needed for that, perhaps?

Also I thought the comment not being resolved wouldn't block the PR from getting merged since I don't have write access to make that happen (the merge indicator at the bottom of this PR is green which signifies that).

Reply:
yup, you're right. txt for replies ;)